appveyor.yml

```diff
@@ -89,6 +89,10 @@ build_script:
   - appveyor PushArtifact out/Default/dist.zip
   - appveyor PushArtifact out/Default/chromedriver.zip
   - appveyor PushArtifact out/ffmpeg/ffmpeg.zip
+  - 7z a node_headers.zip out\Default\gen\node_headers
```

Comment:
Can you rely on utility from standard powershell https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.archive/compress-archive?view=powershell-6 to avoid some accidental breakage in future :)

Reply:
@deepak1556 it looks like the version of Powershell we have on our CI vms doesn't support this command: https://ci.appveyor.com/project/electron-bot/electron-ldhmv/builds/26839690